### DISCUSSION TITLE
SQL Anywhere Connection (v11-...)

### DISCUSSION THREAD

**Discussion by Merowinger68** [dbeaver/dbeaver#13423] [Q&A]
I've been trying to access a SQL Anywhere database with DBeaver for days. I have been working with SQL Anywhere since version 11. Basically, I have the jconnect or jdbc support installed when creating the databases.
When I want to connect via Sybase jConnect, I only get the message that the driver architecture does not match or connection refused.
If I want to set up a connection via ODBC, I get the message, 'connection refused' or 'unexpected string....'.

So it would be really nice if I could directly access a SQL Anywhere.

**Comment by uslss** [comment]
http://infocenter.sybase.com/help/index.jsp?topic=/com.sybase.help.sqlanywhere.11.0.1/dbprogramming_en11/pg-jdbc.html
